src/util/file_test.go

```diff
@@ -78,11 +77,12 @@ func TestInitDataDirDefault(t *testing.T) {
 	defaultDataDir := ".skycointestXCAWDAWD232232"
 	home := UserHome()
 	assertDirNotExists(t, filepath.Join(home, defaultDataDir))
-	dir := InitDataDir("")
+	dir := InitDataDir(defaultDataDir)
 	assert.NotEqual(t, dir, "")
 	assert.True(t, strings.HasSuffix(dir, defaultDataDir))
 	assertDirExists(t, dir)
 	os.RemoveAll(dir)
```

Comment:
Maybe we should wrap `os.RemoveAll` for the tests, and make sure that aren't removing `/`, `$HOME`, etc